src/uct/ib/ud/accel/ud_mlx5.c

```diff
@@ -336,6 +336,9 @@ uct_ud_mlx5_ep_put_short(uct_ep_h tl_ep, const void *buffer, unsigned length,
     uct_ud_send_skb_t *skb;
     size_t wqe_size;
 
+    UCT_CHECK_LENGTH(sizeof(*neth) + sizeof(*put_hdr) + length,
```

Comment:
wrong check for put on mlx5
it is checked below on line 369

Reply:
we have the same checks for AM Short

Reply:
ok, it is not real RDMA, then AM check should work.
anyway it seems excessive - this does the same as check on line 369